### PR TITLE
Move getServerURL to developer tools

### PR DESCRIPTION
**Pull request by thefrieddan1**
## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/CIAC-10497)

## Description
Moving getServerURL script to Developer tools

### CODE REVIEW

File: Packs/DeveloperTools/ReleaseNotes/1_3_14.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Scripts
+
+##### New: GetServerInfo
+
+- New: Get version and URL of current XSOAR server.
```
Comment (ShirleyDenkberg) [inline Packs/DeveloperTools/ReleaseNotes/1_3_14.md:6]:
```suggestion
New: Get version and URL of current Cortex XSOAR server.
```

File: Packs/DeveloperTools/Scripts/GetServerInfo/GetServerInfo.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+comment: 'Get version and URL of current XSOAR server.'
```
Comment (ShirleyDenkberg) [inline Packs/DeveloperTools/Scripts/GetServerInfo/GetServerInfo.yml:1]:
```suggestion
comment: Get version and URL of current XSOAR server.
```

File: Packs/DeveloperTools/Scripts/GetServerInfo/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+Get version and URL of current XSOAR server.
```
Comment (ShirleyDenkberg) [inline Packs/DeveloperTools/Scripts/GetServerInfo/README.md:1]:
```suggestion
Get version and URL of current Cortex XSOAR server.
```

File: Packs/DeveloperTools/Scripts/GetServerInfo/README.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+---
+There are no inputs for this script.
+
+## Outputs
+
+---
+
+| **Path** | **Description** | **Type** |
+| --- | --- | --- |
+| ServerInfo | A dictionary of "urls" and "version" | Dictionary |
```
Comment (ShirleyDenkberg) [inline Packs/DeveloperTools/Scripts/GetServerInfo/README.md:24]:
```suggestion
| ServerInfo | A dictionary of "URLs" and "version". | Dictionary |
```

File: Packs/DeveloperTools/Scripts/GetServerInfo/GetServerInfo.yml [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+commonfields:
+  id: GetServerInfo
+  version: -1
+name: GetServerInfo
+outputs:
+- contextPath: ServerInfo
+  description: 'A dictionary of "urls" and "version".'
```
Comment (ShirleyDenkberg) [inline Packs/DeveloperTools/Scripts/GetServerInfo/GetServerInfo.yml:8]:
```suggestion
  description: 'A dictionary of "URLs" and "version".'
```

**Comment by ShirleyDenkberg** [comment]
@thefrieddan1 Doc review completed.

**Review by JudahSchwartz — APPROVED** [review]

**Pull request merged**